packages/core/bootstrap/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
```

Comment:
Please add some info about this to README, and possibly a package.json script shortcut.